
2025-09-26 22:24:10,469 [INFO] __main__: –ó–∞–ø—É—Å–∫ —Å–µ—Ä–≤–µ—Ä–∞ –Ω–∞ http://0.0.0.0:8000/docs
2025-09-26 22:24:24,007 [INFO] backend.routes: –ü–æ–ª—É—á–µ–Ω –∑–∞–ø—Ä–æ—Å –Ω–∞ –∞–Ω–∞–ª–∏–∑: 1 –æ—Ç–∑—ã–≤–æ–≤
2025-09-26 22:29:51,703 [INFO] __main__: –ó–∞–ø—É—Å–∫ —Å–µ—Ä–≤–µ—Ä–∞ –Ω–∞ http://0.0.0.0:8000/docs
2025-09-26 22:29:57,712 [INFO] backend.routes: –ü–æ–ª—É—á–µ–Ω –∑–∞–ø—Ä–æ—Å –Ω–∞ –∞–Ω–∞–ª–∏–∑: 1 –æ—Ç–∑—ã–≤–æ–≤
2025-09-26 22:30:31,394 [INFO] __main__: –ó–∞–ø—É—Å–∫ —Å–µ—Ä–≤–µ—Ä–∞ –Ω–∞ http://0.0.0.0:8000/docs
2025-09-26 22:30:39,231 [INFO] backend.routes: –ü–æ–ª—É—á–µ–Ω –∑–∞–ø—Ä–æ—Å –Ω–∞ –∞–Ω–∞–ª–∏–∑: 1 –æ—Ç–∑—ã–≤–æ–≤
2025-09-26 22:35:38,852 [INFO] __main__: –ó–∞–ø—É—Å–∫ —Å–µ—Ä–≤–µ—Ä–∞ –Ω–∞ http://0.0.0.0:8000/docs
2025-09-26 22:35:46,099 [INFO] backend.routes: –ü–æ–ª—É—á–µ–Ω –∑–∞–ø—Ä–æ—Å –Ω–∞ –∞–Ω–∞–ª–∏–∑: 1 –æ—Ç–∑—ã–≤–æ–≤
2025-09-26 22:37:14,550 [INFO] __main__: –ó–∞–ø—É—Å–∫ —Å–µ—Ä–≤–µ—Ä–∞ –Ω–∞ http://0.0.0.0:8000/docs
2025-09-26 22:37:17,137 [INFO] utils.time_util: üîÑ [ASYNC] –í—ã–ø–æ–ª–Ω—è–µ—Ç—Å—è 'analyze'...
2025-09-26 22:37:17,137 [INFO] backend.routes: –ü–æ–ª—É—á–µ–Ω –∑–∞–ø—Ä–æ—Å –Ω–∞ –∞–Ω–∞–ª–∏–∑: 1 –æ—Ç–∑—ã–≤–æ–≤
2025-09-26 22:37:18,516 [INFO] utils.time_util: ‚úÖ [ASYNC] 'analyze' –∑–∞–≤–µ—Ä—à—ë–Ω –∑–∞ 1.38 —Å–µ–∫.
2025-09-26 22:38:48,950 [INFO] __main__: –ó–∞–ø—É—Å–∫ —Å–µ—Ä–≤–µ—Ä–∞ –Ω–∞ http://0.0.0.0:8000/docs
2025-09-26 22:51:56,560 [INFO] backend.routes: –ü–æ–ª—É—á–µ–Ω –∑–∞–ø—Ä–æ—Å –Ω–∞ –∞–Ω–∞–ª–∏–∑: 10 –æ—Ç–∑—ã–≤–æ–≤
2025-09-26 22:52:00,704 [INFO] utils.time_util: ‚úÖ [ASYNC] 'analyze' –∑–∞–≤–µ—Ä—à—ë–Ω –∑–∞ 4.14 —Å–µ–∫.
2025-09-26 23:21:54,514 [INFO] __main__: –ó–∞–ø—É—Å–∫ —Å–µ—Ä–≤–µ—Ä–∞ –Ω–∞ http://0.0.0.0:8000/docs
2025-09-26 23:22:05,108 [INFO] backend.routes: –ü–æ–ª—É—á–µ–Ω –∑–∞–ø—Ä–æ—Å –Ω–∞ –∞–Ω–∞–ª–∏–∑: 10 –æ—Ç–∑—ã–≤–æ–≤
2025-09-26 23:22:09,009 [ERROR] backend.routes: –ù–µ–∏–∑–≤–µ—Å—Ç–Ω–∞—è –æ—à–∏–±–∫–∞ –ø—Ä–∏ –æ–±—Ä–∞–±–æ—Ç–∫–µ: BaseModel.model_dump() got an unexpected keyword argument 'utf8'
Traceback (most recent call last):
  File "/work/CustomerSentimentChecker/api/backend/routes.py", line 37, in analyze
    result = processor.process_batch_threads(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/work/CustomerSentimentChecker/api/processor/review_processor.py", line 71, in process_batch_threads
    return self.formatter.format_output(results)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/work/CustomerSentimentChecker/api/processor/json_formatter.py", line 90, in format_output
    return output_data.model_dump(exclude_none=True, utf8=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: BaseModel.model_dump() got an unexpected keyword argument 'utf8'
2025-09-26 23:22:09,016 [ERROR] utils.time_util: ‚ùå [ASYNC] –û—à–∏–±–∫–∞ –≤ 'analyze' –∑–∞ 3.91 —Å–µ–∫.: HTTPException: 500: Internal Server Error
2025-09-26 23:22:40,187 [INFO] __main__: –ó–∞–ø—É—Å–∫ —Å–µ—Ä–≤–µ—Ä–∞ –Ω–∞ http://0.0.0.0:8000/docs
2025-09-26 23:22:43,654 [INFO] backend.routes: –ü–æ–ª—É—á–µ–Ω –∑–∞–ø—Ä–æ—Å –Ω–∞ –∞–Ω–∞–ª–∏–∑: 10 –æ—Ç–∑—ã–≤–æ–≤
2025-09-26 23:22:47,687 [INFO] utils.time_util: ‚úÖ [ASYNC] 'analyze' –∑–∞–≤–µ—Ä—à—ë–Ω –∑–∞ 4.03 —Å–µ–∫.
